middlewares/userSchemas.js

```diff
@@ -0,0 +1,22 @@
+const Joi = require('joi');
+
+const emailRegEx =
+  /^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?$/;
+
+const registerSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().email().required(),
+  password: Joi.string().min(6).required(),
+});
+
+const loginSchema = Joi.object({
+  email: Joi.string().pattern(emailRegEx).required(),
```

Comment:
```suggestion
  email: Joi.string().email().required(),
```
можна також використати метод email. Чи це ви для розмаїття?)

Reply:
Заради цікавості)